# time in miliseconds
uint32 timestamp
# delta x in meters
float32 dx
# delta y in meters
float32 dy
# absolute z in meters
float32 z 
# absolute yaw angle in [-180, 180] degrees
float32 yaw_deg
# yaw rate in [-180, 180] degrees
float32 yaw_rate_deg
# absolute ground truth source direction in [-180, 180] degrees
float32 source_direction_deg
